test/manual/socks5.test.js

```diff
@@ -0,0 +1,325 @@
+'use strict';
+const { MongoClient } = require('../../src');
```

Comment:
We've been moving tests to TS files, I tried it locally, and it looks like you would just have to change these to es imports if you wanna make that improvement.